src/sentry/shared_integrations/exceptions/__init__.py

```diff
@@ -64,7 +71,7 @@ class DuplicateDisplayNameError(IntegrationError):
 
 
 class IntegrationFormError(IntegrationError):
-    def __init__(self, field_errors):
+    def __init__(self, field_errors: Sequence[Exception]):
```

Comment:
Isn't this a `Mapping[str, Exception]`? It seems like that's what `error_fields_from_json` returns anyway